### ISSUE TITLE
Arc A770 Can't use on WSL2(Ubuntu24.04LTS)

### ISSUE THREAD

**Issue by Vaccummer** [intel/intel-extension-for-pytorch#761]
### Describe the bug

I encounter the error on WSL2(Ubuntu24.04LTS) with Arc A770, 14700K。
(arc) am@DESKTOP-0M98CHE:~/codes/AIF/CV-Starter$ python -c "import torch; import intel_extension_for_pytorch as ipex; print(torch.__version__); print(ipex.__version__); [print(f'[{i}]: {torch.xpu.get_device_properties(i)}') for i in range(torch.xpu.device_count())];"
[W102 23:42:19.647386805 OperatorEntry.cpp:155] Warning: Warning only once for all operators,  other operators may also be overridden.
  Overriding a previously registered kernel for the same operator and the same dispatch key
  operator: aten::_cummax_helper(Tensor self, Tensor(a!) values, Tensor(b!) indices, int dim) -> ()
    registered at /build/pytorch/build/aten/src/ATen/RegisterSchema.cpp:6
  dispatch key: XPU
  previous kernel: registered at /build/pytorch/build/aten/src/ATen/RegisterCPU.cpp:30476
       new kernel: registered at /build/intel-pytorch-extension/build/Release/csrc/gpu/csrc/aten/generated/ATen/RegisterXPU.cpp:2971 (function operator())
[2025-01-02 23:42:19,435] [INFO] [real_accelerator.py:219:get_accelerator] Setting ds_accelerator to xpu (auto detect)
Abort was called at 63 line in file:
./shared/source/os_interface/os_interface.h
Aborted (core dumped)

### Versions
14700k
Arc A700
Win10 WSL2(Ubuntu24.04LTS)
driver:gfx_win_101.6314, I also install driver in WSL2

**Comment by ZailiWang** [comment]
Hi, would you double confirm if you have followed [Driver installation on Ubuntu 24.04](https://dgpu-docs.intel.com/driver/client/overview.html#installing-client-gpus-on-ubuntu-desktop-24-04-lts)?

**Comment by Vaccummer** [comment]
> Hi, would you double confirm if you have followed [Driver installation on Ubuntu 24.04](https://dgpu-docs.intel.com/driver/client/overview.html#installing-client-gpus-on-ubuntu-desktop-24-04-lts)?

I obey the instruction on https://dgpu-docs.intel.com/driver/client/overview.html, and I can verify my installation:
(arc) am@DESKTOP-0M98CHE:~$ clinfo | grep "Device Name"
  Device Name                                     Intel(R) Graphics [0x56a0]
  Device Name                                     Intel(R) Graphics [0xa780]
    Device Name                                   Intel(R) Graphics [0x56a0]
    Device Name                                   Intel(R) Graphics [0x56a0]
    Device Name                                   Intel(R) Graphics [0x56a0]

**Comment by ZailiWang** [comment]
Would you help verify if disabling Raptor Lake iGPU (0xa780) could solve the issue? This can be done in "Device Manager" in Win system.

**Comment by Vaccummer** [comment]
> Would you help verify if disabling Raptor Lake iGPU (0xa780) could solve the issue? This can be done in "Device Manager" in Win system.

Solved! Thanks for your help!
